command/intake.py

```diff
@@ -0,0 +1,50 @@
+import config
+import wpilib
+from robotpy_toolkit_7407 import SubsystemCommand
+from oi.keymap import Controllers
+from subsystem import Intake
+from robotpy_toolkit_7407.utils.units import radians
+
+
+class SetIntake(SubsystemCommand[Intake]):
+    def __init__(self, subsystem: Intake, intake_active: bool, go_cube: bool, go_cone: bool):
```

Comment:
also when you implement the wrist functionality, make sure to add it to this command and not create a new one, since we can only run one command per subsystem at a time